app/views/pages/contact.html.haml

```diff
@@ -0,0 +1,4 @@
+%section#contact.container
+  - provide(:title, 'Contact')
+  %h1 Contact
+  %p=t('contact_intro_html', about: (link_to "About", page_path('about')))
```

Comment:
Files should end with a trailing newline<br>The = symbol should have one space separating it from code